covsirphy/cleaning/jhu_complement.py

```diff
@@ -25,6 +25,14 @@ class JHUDataComplementHandler(Term):
         Status code: 'fully complemented recovered data' and so on as noted in self.run() docstring.
     """
     RAW_COLS = [Term.C, Term.F, Term.R]
+    MONOTONIC_CONFIRMED = "Monotonic_confirmed"
+    MONOTONIC_FATAL = "Monotonic_fatal"
+    MONOTONIC_RECOVERED = "Monotonic_recovered"
+    FULL_RECOVERED = "Full_recovered"
+    PARTIAL_RECOVERED = "Partial_recovered"
+    RECOVERED_COLS = [MONOTONIC_RECOVERED, FULL_RECOVERED, PARTIAL_RECOVERED]
+    SHOW_COMPLEMENT_COLS = [Term.C, Term.F, *RECOVERED_COLS]
+    SHOW_COMPLEMENT_FULL_COLS = [MONOTONIC_CONFIRMED, MONOTONIC_FATAL, *RECOVERED_COLS]
```

Comment:
We have `SHOW_COMPLEMENT_COLS` and `SHOW_COMPLEMENT_FULL_COLS`. Which list will be used in this new feature?